src/Components/Nav/Nav.js

```diff
@@ -14,9 +14,20 @@ const controlActivation = [
 function Nav() {
   const [menu, setMenu] = useState(1);
   const history = useHistory();
+  const { pathname } = useLocation();
+  const isNavActive = !(
+    pathname === "/" ||
+    pathname === "/login" ||
+    pathname === "/signup" ||
+    pathname === "/phone_validate" ||
+    pathname === "/set_account" ||
+    pathname === "/subscribe" ||
+    pathname === "/payments" ||
+    pathname === "/category"
+  );
```

Comment:
some 메서드를 활용하면 다음과 같이 할 수 있을 것 같습니다!
```suggestion
  const pathList = ["/","/login","/signup","/phone_validate","/set_account", "/payments", "/category"];
  const isNavActive = !pathList.some(currentPath => pathname === currentPath)
```

Reply:
수정했습니다! 🙇‍♀️